### ISSUE TITLE
resource/aws_ecs_service: Intermittent acceptance test failure in TestAccAWSEcsService_withLaunchTypeFargateAndPlatformVersion

### ISSUE THREAD

**Issue by gdavison** [hashicorp/terraform-provider-aws#18364]
There is a rare intermittent failure in `TestAccAWSEcsService_withLaunchTypeFargateAndPlatformVersion` with the following error:

```console
=== RUN   TestAccAWSEcsService_withLaunchTypeFargateAndPlatformVersion
=== PAUSE TestAccAWSEcsService_withLaunchTypeFargateAndPlatformVersion
=== CONT  TestAccAWSEcsService_withLaunchTypeFargateAndPlatformVersion
testing_new.go:63: Error running post-test destroy, there may be dangling resources: exit status 1
Error: Error deleting security group: DependencyViolation: resource sg-0467142aaec35c0a4 has a dependent object
  status code: 400, request id: e9327a0c-d814-47e4-bb51-e69892ab834a
Error: Error deleting security group: DependencyViolation: resource sg-093b061548736cdec has a dependent object
  status code: 400, request id: 7677f1c7-6101-467d-95de-4374b2ca385f
--- FAIL: TestAccAWSEcsService_withLaunchTypeFargateAndPlatformVersion (794.34s)
```

### Affected Resource

* aws_ecs_service

**Comment by YakDriver** [comment]
I've been noticing this also. Strange...

**Comment by YakDriver** [comment]
I can't guarantee 100% that this problem has been totally fixed. However, I was regularly gettting it before but with the fixes in #16936 I haven't seen it once. I've ran these tests probably 10X to duplicate without seeing it. Please let us know if you see this issue post v3.35.0. Otherwise, it's fairly safe to say that we got it.l
